src/otx/recipe/action/action_detection/x3d_fastrcnn.yaml

```diff
@@ -0,0 +1,93 @@
+model:
+  class_path: otx.algo.action_detection.x3d_fastrcnn.X3DFastRCNN
+  init_args:
+    num_classes: 81
+    topk: 3
+
+optimizer:
+  class_path: torch.optim.SGD
+  init_args:
+    lr: 0.005
+    momentum: 0.9
+    weight_decay: 0.00001
+
+scheduler:
+  class_path: lightning.pytorch.cli.ReduceLROnPlateau
+  init_args:
+    mode: min
+    factor: 0.1
+    patience: 10
+    monitor: train/loss
+
+engine:
+  task: ACTION_DETECTION
+  device: auto
```

Comment:
Yes, but at the same time, it can be used by declaring it in places like `train.yaml` (which is linked to `engine.train()`). In my opinion, this could be confusing for users using configuration, so I didn't add it to engine.

Reply:
that's fair, we should make it clear  when we write the documentation that it is also possible. Let's keep that in mind when the time comes to write the documentation